lib/sufia/export/converter.rb

```diff
@@ -0,0 +1,21 @@
+module Sufia
+  module Export
+    # a base class to convert an ActiveFedora object that contains permissions and allow for pretty json.
+    #
+    class Converter
+      # overrides to_json to optionally allow for a pretty version of the json to be outputted
+      #
+      # @param [Boolean] pretty pass true to output formatted json using pretty_generate
+      def to_json(pretty = false)
+        return super unless pretty
+        JSON.pretty_generate(JSON.parse(to_json))
```

Comment:
I would go for `super()` to remove the recursive call.

I think `super` is a keyword whereas `super()` is a function call. Ruby might not like the keyword inside another function call.